tests/system_tests/conftest.py

```diff
@@ -26,6 +21,9 @@
 )
 from .wandb_backend_spy import WandbBackendProxy, WandbBackendSpy, spy_proxy
 
+#: See https://docs.pytest.org/en/stable/how-to/plugins.html#requiring-loading-plugins-in-a-test-module-or-conftest-file
+pytest_plugins = ("tests.system_tests.backend_fixtures",)
+
```

Comment:
**note:** This loads `backend_fixtures.py` as a pytest plugin, following guidance from the pytest [docs](https://docs.pytest.org/en/stable/how-to/plugins.html#requiring-loading-plugins-in-a-test-module-or-conftest-file)